langchain4j-core/src/main/java/dev/langchain4j/store/embedding/IngestionResult.java

```diff
@@ -0,0 +1,46 @@
+package dev.langchain4j.store.embedding;
+
+import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.model.output.TokenUsage;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents the result of a {@link EmbeddingStoreIngestor} ingest process.
+ */
+public class IngestionResult {
+    /**
+     * The embeddings content that were ingested.
+     */
+    private final List<Embedding> content;
```

Comment:
This PR is about tracking token usage. Why are embeddings and metadata included? Is this reqlly required? I would rather remove them and return only `TokenUsage`. We can always add other things later, but it does not seem to be very useful right now.

Reply:
Sure! Makes sense! fixed!